### ISSUE TITLE
Bangle 1 firmware update instructions missing

### ISSUE THREAD

**Issue by timp** [espruino/EspruinoDocs#737]
https://www.espruino.com/Firmware+Update#nrf52 does not give guidance on updating the original Bangle 1 firmware.

**Comment by timp** [comment]
Web IDE Flasher worked, but the modal does  not tell you it can be closed when the progress bar is complete.

**Comment by gfwilliams** [comment]
There do seem to be instructions under `Entering DFU mode` along with the other boards?

![image](https://github.com/user-attachments/assets/3e62d54b-ba9a-4de4-b809-4f008a8e95b6)

Is your issue that Bangle.js 1 isn't specifically called out at the top? The reason Bangle.js 2 is called out is because usually we want users to update the firmware a different way on that.

> but the modal does not tell you it can be closed when the progress bar is complete.

I've just checked and the modal should be closed automatically, but I know on Bangle.js 1 I have noticed times where the update gets to 99% and doesn't signal completion (even though it's fine), so that could be it. It shouldn't happen all the time though.
